ui-core/src/participant/ParticipantNavbar.tsx

```diff
@@ -398,6 +398,11 @@ export const AccountOptionsDropdown = (
               {i18n('manageProfiles')}
             </button>
           </NavLink>}
+          <NavLink to="/hub/kits">
+            <button className="dropdown-item" aria-label="sample kits">
+              Sample Kits
+            </button>
+          </NavLink>
```

Comment:
Should this be hidden behind study configuration settings? Right now we don't have a "turn kits on and off" but maybe we should introduce that setting alongside the in person kits setting.

Reply:
concur -- this could confuse HeartHive folks if they see a "no sample kits" message, even though they have done a kit just not through Juniper

Reply:
That's a great callout- I just updated the dropdown to only display the "Sample Kits" button if at least one kit type has been configured for the given study environment